modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -1039,6 +1046,8 @@ def _resample_func(
             Modin frame. If not specified will be computed automaticly.
         df_op : callable(pandas.DataFrame) -> [pandas.DataFrame, pandas.Series], optional
             Preprocessor function to apply to the passed frame before resampling.
+        allow_range_impl : bool, default: True
```

Comment:
```suggestion
        range_impl : bool, default: True
```
or use_range_impl?

Reply:
`allow_range_impl=True` doesn't necessarily mean, that the range-partition will be used, it also depends on `cfg.RangePartitioning` value and [`axis` argument](https://github.com/dchigarev/modin/blob/bc1fc691ad1968da8640a8e5416bed857bedad18/modin/core/storage_formats/pandas/query_compiler.py#L1119-L1122). So this parameter indeed only 'allows' for the range-partitioning to be used, not dictates that.